src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -155,6 +158,17 @@ public Errors validatePartialUpdateDetails(OverseasEntitySubmissionDto overseasE
         return errors;
     }
 
+    private Errors validatePartialRemoveDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+
+        errors = validatePartialCommonDetails(overseasEntitySubmissionDto, errors, loggingContext);
+
+        if (overseasEntitySubmissionDto.getRemove() != null) {
```

Comment:
Instead of calling overseasEntitySubmissionDto.getRemove() twice, you can use a var instead:
` 
var removeDto = overseasEntitySubmissionDto.getRemove();
if (removeDto != null) {
   removeValidator.validate(removeDto, errors, loggingContext);
}
`

Reply:
use of var added